arches/app/views/plugin.py

```diff
@@ -23,8 +23,11 @@
 from arches.app.utils.betterJSONSerializer import JSONSerializer
 from arches.app.utils.response import JSONResponse
 from arches.app.views.base import MapBaseManagerView
+from django.utils.decorators import method_decorator
+from arches.app.utils.decorators import login_required
 
 
+# @method_decorator(login_required, name="dispatch")
```

Comment:
@RobOatesHistoricEngland - If it's commented out, remove?

Reply:
Was testing, but the decorator should be there, not commented-out. Have pushed the change.